{
    "type": "bundle",
    "id": "bundle--03a38864-07b9-482b-b50a-3605147423f3",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--c3c7a857-d4a2-40c9-8196-ba30dc138e61",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2023-05-06T00:04:42.832819Z",
            "modified": "2023-05-06T00:04:42.832819Z",
            "name": "CVE-2023-29935",
            "description": "llvm-project commit a0138390 was discovered to contain an assertion failure at !replacements.count(op) && \"operation was already replaced.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2023-29935"
                }
            ]
        }
    ]
}